src/test/java/com/scaleunlimited/flinkcrawler/metrics/CrawlerMetricsTest.java

```diff
@@ -0,0 +1,92 @@
+package com.scaleunlimited.flinkcrawler.metrics;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.metrics.Metric;
+import org.apache.flink.metrics.MetricConfig;
+import org.apache.flink.metrics.MetricGroup;
+import org.apache.flink.metrics.reporter.MetricReporter;
+import org.apache.flink.runtime.client.JobExecutionException;
+import org.apache.flink.streaming.api.datastream.DataStreamSource;
+import org.apache.flink.streaming.api.environment.LocalStreamEnvironment;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.scaleunlimited.flinkcrawler.functions.BaseMapFunction;
+
+public class CrawlerMetricsTest {
+    
+    public static class TestReporter implements MetricReporter {
+        
+        private static Map<String, Metric> _metrics;
```

Comment:
If it's static, `METRICS` would be more obvious.

Reply:
This file was inadvertently committed, and I've since removed it. I'll take your suggestions into account when I commit this to my other branch, though.